.ci/pipeline_definitions

```diff
@@ -104,16 +104,16 @@ vertical-pod-autoscaler:
         dockerimages:
           vpa-recommender:
             image: europe-docker.pkg.dev/gardener-project/snapshots/gardener/autoscaler/vertical-pod-autoscaler/vpa-recommender
-            dockerfile: 'Dockerfile'
-            dir: 'vertical-pod-autoscaler/pkg/recommender'
+            dockerfile: './pkg/recommender/Dockerfile'
```

Comment:
Shoultn't we use

```suggestion
            dockerfile: 'pkg/recommender/Dockerfile'
```

Reply:
Checking whether this will work at all with @AndreasBurger

Reply:
Done.